static/js/components/UserManagement.jsx

```diff
@@ -449,6 +547,94 @@ const UserManagement = () => {
       <br />
       {invitationsEnabled && (
         <>
+          {!!invitations?.length && (
+            <>
+              <Typography variant="h5">Pending Invitations</Typography>
+              <Paper elevation={1}>
```

Comment:
```suggestion
              <Paper elevation={1} className={classes.invitations}>
```
I would add a corresponding styling class at the top of the file to add a marginBottom to this paper (right now the bulk invite header is right up against the pending invitations table).

Something like this:
`  invitations: {
    marginBottom: "1rem",
  },`

Reply:
👍